source/configuration_manual/plugins/acl.rst

```diff
@@ -40,6 +40,9 @@ ACLs can be enabled in ``dovecot.conf`` with:
     # This reduces unnecessary disk I/O when only global ACLs are used. (v2.2.31+)
     #acl_globals_only = yes
 
+    # Namespace(s) to ignore
+    #acl_ignore_namespace =
```

Comment:
Maybe: `Namespace prefix to ignore. Use counter to ignore multiple, e.g. acl_ignore_namespaces2`